DEFAULT_CONFIG.json5

```diff
@@ -326,11 +326,20 @@
       },
     },
     link: {
-      /// An optional whitelist of protocols to be used for accepting and opening sessions.
-      /// If not configured, all the supported protocols are automatically whitelisted.
-      /// The supported protocols are: ["tcp" , "udp", "tls", "quic", "ws", "unixsock-stream", "vsock"]
-      /// For example, to only enable "tls" and "quic":
-      //   protocols: ["tls", "quic"],
+      /// An optional whitelist of protocols to be used for accepting and opening sessions. If not
+      /// configured, all the supported protocols are automatically whitelisted. The supported
+      /// protocols are: ["tcp" , "udp", "tls", "quic", "ws", "unixsock-stream", "vsock"] For
+      /// example, to only enable "tls" and "quic": protocols: ["tls", "quic"],
+      ///
+      /// ## Endpoint metadata
```

Comment:
Yes, but it would need to be duplicated. Is it worth it?

Reply:
I don't think we should have this section at all. We should configure this kind of metadata via the `Endpoint` metadata syntax and not in the config.

Reply:
I'm not sure I understand your comment.

>  We should configure this kind of metadata via the Endpoint metadata syntax and not in the config.

Are you suggesting to move this section to the documentation of `Endpoint`?

Reply:
Yes, and potentially move it just before `connect`/`listen` section because it's there where the user will set the those metadata. Moreover, it would be good to provide some example (either on `Endpoint` or in the config)

Reply:
Yes, nothing against it.